sdk/storage/azure-storage-file/HISTORY.md

```diff
@@ -1,6 +1,24 @@
 # Change Log azure-storage-file
 
 
+## Version 12.0.0b3:
+
+**New features**
+- Added FileSmbProperties REST Parity.
+- Added support for AppendBlockFromURL, PutPageFromURL, PutRangeThroughURL.
```

Comment:
AppendBlockFromURL, PutPageFromURL are for blob.

Reply:
oops